e2e-demo/image-builder/rhel-85.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
@fzdarsky This file is adjusted based on the official composer rhel-85.json. Therefore will not create a symbolic link to the other file.